photon-client/src/components/settings/ApriltagControlCard.vue

```diff
@@ -0,0 +1,95 @@
+<script setup lang="ts">
+import CvSlider from "@/components/common/cv-slider.vue";
+import { useCameraSettingsStore } from "@/stores/settings/CameraSettingsStore";
+import { useStateStore } from "@/stores/StateStore";
+import { useSettingsStore } from "@/stores/settings/GeneralSettingsStore";
+import { Euler, Quaternion } from "three";
+
+function quatToEuler(quat) {
+  console.log(quat);
+  quat = new Quaternion(quat.X, quat.Y, quat.Z, quat.W);
+  return new Euler().setFromQuaternion(quat, "ZYX");
+}
+
+// Convert from radians to degrees.
+const degrees = function (radians) {
```

Comment:
Add type `number `for `radians `and (nit) convert this to arrow function declaration for convention